demostore_automation/tests/frontend/home_page/test_dropdown_menu.py

```diff
@@ -10,5 +10,33 @@ class TestDropDown:
     Test class to verify the display of sorting drop-down on the home page
     """
 
-    @pytest.fixture ('scoop='class')
-    
\ No newline at end of file
+    @pytest.fixture (scoop='class')
```

Comment:
I dont think this causes problem but it looks weird having space before the (. `@pytest.fixture(scope='class')` vs `@pytest.fixture (scope='class')`.
But also spelling. It should be `scope` not `scoop`. I would've expected this to fail.
Did you run the test?